package.json

```diff
@@ -80,6 +80,7 @@
     "doctoc/**/trim": "^1.0.0",
     "postcss": "^8.2.1",
     "parcel-bundler/cssnano": "^5.0.2",
+    "browserslist": "^4.16.5",
```

Comment:
I think it's best to keep unrelated changes in separate PRs, otherwise the commit message is misleading

Reply:
I generally agree. But CI was failing for this PR. For fixing vulnerability updates, should we merge even if CI is failing and then submit a follow-up PR?